src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -73,22 +73,53 @@ final class WP_Interactivity_API {
 	 */
 	private $has_processed_router_region = false;
 
+	/**
+	 * Stack of namespaces defined by `data-wp-interactive` directives, in
+	 * the order they are processed.
+	 *
+	 * This list is populated only during directive processing, and it should
+	 * remain empty afterwards.
+	 *
+	 * @since 6.6.0
+	 * @var array
+	 */
+	private $namespace_stack = array();
+
+	/**
+	 * Stack of contexts defined by `data-wp-context` directives, in
+	 * the order they are processed.
+	 *
+	 * This list is populated only during directive processing, and it should
+	 * remain empty afterwards.
+	 *
+	 * @since 6.6.0
+	 * @var array
```

Comment:
These are the actual contexts, right? They'll be arrays?
```suggestion
	 * @var array<array<mixed>>
```